Makefile

```diff
@@ -62,6 +62,7 @@ $(DIST):
 	mkdir -p $(DISTDIR)
 	npm install || (cat npm-debug.log && exit 1)
 	npm run build
+	npm install --production || (cat npm-debug.log && exit 1)
```

Comment:
OK, I think I found why it's not working. We lack npm-shrinkwrap.json.

It seems that only with npm-shrinkwrap.json being present `npm install` followed by `npm install --production` actually removes dev dependencies.